packages/UI/src/components/ShareDialog/index.ts

```diff
@@ -0,0 +1,2 @@
+export { ShareDialog as ShareDialog } from './ShareDialog';
```

Comment:
You can do `export { ShareDialog } from './ShareDialog';`
no need for the `as` if the name is the same